regolith/tools.py

```diff
@@ -2133,3 +2135,65 @@ def get_tags(coll):
     all_tags = list(set(all_tags))
     all_tags.sort()
     return all_tags
+
+
+# create Google Doc page for prum in Google Drive 'bg-projects'
+def gdoc_generator(rc):
+    """Create Google Doc for new projectum in 'generated_projecta_docs' folder in 'bg-projects' Google Drive."""
```

Comment:
since we are moving this to tools it should be more general, so this is not a "projectum" specific function but a function that creates a gdoc for any helper to use.